script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -304,27 +323,27 @@ That's...
 -- Page 587, Offset 7144.
 -- 微妙に違ってはいたが、この眼鏡をくれた先生が、子供だった俺に教えてくれた事に近かった。
 // Association of receiving the glasses changed to help flow
-...A little different from my understanding, but it's close to what Sensei taught me when I received these glasses from her as a child.
+A little different from my understanding, but it's close to what Sensei taught me when I received these glasses from her as a child.
 }
 [sha:b5198fa9673acec4f50067ed1e27a47f246b1269]{
 -- Page 588, Offset 7145.
 -- ……万物の結果。ものの未来。
-...The end of all things. The future of things.
+The end of all things. The future of an object.
```

Comment:
Revert this and the following two lines, lore reasons. もの here also refers to [all] things. What Shiki has been unconsciously avoiding is the fact that his eyes have this power.

Reply:
Oops yeah misread on the second two a bit. Though, it does seem to me like the object is the end/future rather than his eyes themselves, even though the two are obviously connected? I guess I really just want to change 'that fact' because it feels like a weird way to apply back.
> The end of all things. The future of things.
> Without even realizing what I was doing―――
> I've spent all this time desperately trying not to be conscious of them.

Reply:
works for me